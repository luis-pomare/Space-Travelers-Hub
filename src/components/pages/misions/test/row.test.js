import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';

import Row from '../row';

const Element = () => {
  <BrowserRouter>
    <Row />
  </BrowserRouter>;
};

it('Renders ok', () => {
  const content = renderer.create(<Element />).toJSON();
  expect(content).toMatchSnapshot();
});
