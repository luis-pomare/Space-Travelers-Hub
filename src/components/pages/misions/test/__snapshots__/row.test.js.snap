// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders ok 1`] = `null`;
